rlclientlib/factory_resolver.cc

```diff
@@ -68,6 +68,13 @@ namespace reinforcement_learning {
     return error_code::success;
   }
 
+  template <>
+  int model_create<m::vw_model>(m::i_model** retval, const u::configuration& c, i_trace* trace_logger, api_status* status)
```

Comment:
Why do we need this explicit special-case? It looks identical to the generic template above it.